src/components/missions/MissionsList.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable camelcase */
+import { useSelector, useDispatch } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { useEffect } from 'react';
+import { Table } from 'react-bootstrap';
+import MissionItem from './MissionItem';
+import style from './missions.module.css';
+import { fetchGetMissions } from '../../redux/missions/missions';
+
+const Missions = () => {
+  const dispatch = useDispatch();
+  const { missions } = useSelector((state) => state.missions);
+  const loadAction = bindActionCreators(fetchGetMissions, dispatch);
+  useEffect(() => {
+    if (missions.length === 0) dispatch(loadAction);
+  }, []);
+
+  return (
+    <div className={style.mainDiv}>
+      <Table striped bordered className={style.test}>
+        <tbody>
+          <tr>
+            <th>Mission</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th>.</th>
```

Comment:
- [ ] Please change this column name to something more meaningful like "Action" or something that you may like to have a more descriptive table.

Reply:
Done ✔